contract/src/lib.rs

```diff
@@ -1,19 +1,70 @@
 use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
 use near_sdk::collections::{LookupMap, UnorderedSet};
-use near_sdk::{env, near_bindgen, CryptoHash, Promise};
-use near_sdk::{AccountId, BorshStorageKey};
+use near_sdk::{env, near_bindgen, AccountId, BorshStorageKey, CryptoHash, Promise};
+use serde::ser::{Serialize, SerializeStruct, Serializer};
+use std::collections::hash_map::DefaultHasher;
+use std::hash::{Hash, Hasher};
+
+type ContextId = String;
+
+#[derive(BorshSerialize, BorshDeserialize)]
+pub struct PaidContent {
+    id: String,
+    link: String,
+    author: AccountId,
+    cost: String,
+    context_id: ContextId,
+}
+
+// This is what #[derive(Serialize)] would generate.
+impl Serialize for PaidContent {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        let mut s = serializer.serialize_struct("PaidContent", 5)?;
+        s.serialize_field("id", &self.id)?;
+        s.serialize_field("link", &self.link)?;
+        s.serialize_field("author", &self.author)?;
+        s.serialize_field("cost", &self.cost)?;
+        s.serialize_field("context_id", &self.context_id)?;
+        s.end()
+    }
+}
```

Comment:
it is not necessary. 
`#[near_bindgen]` should generate it

Reply:
fixed